docs/operators/ftpInputOperator.md

```diff
@@ -0,0 +1,54 @@
+FTP Input Operator
+=============
+
+## Operator Objective
+This operator is designed to scan a directory from an FTP server for files, read
+and split file content into tuples such as lines or blocks of bytes, and finally
+emit them on the output port for further processing by downstream operators.
+The operator extends the AbstractFileInputOperator. It overrides the
+getFSInstance() method and returns an instance of the FTPFileSystem
+(org.apache.hadoop.fs.ftp.FTPFileSystem)
+
+## Class Diagram
+![FTPInputOperator class diagram](images/ftpInputOperator/classdiagram.png)
```

Comment:
Please show the complete class diagram including all the interfaces implemented in the base class.